joblib/hashing.py

```diff
@@ -93,39 +88,7 @@ def memoize(self, obj):
             return
         Pickler.memoize(self, obj)
 
-    # The dispatch table of the pickler is not accessible in Python
-    # 3, as these lines are only bugware for IPython, we skip them.
-    def save_global(self, obj, name=None, pack=struct.pack):
```

Comment:
I remember this was causing dill and numpypickle integration to fail but forgot the exact reason. I think back then the ipython pickle worked with this removed but I can check again.